bundle/manifests/observability-operator.clusterserviceversion.yaml

```diff
@@ -88,8 +92,23 @@ spec:
     - kind: ThanosRuler
       name: thanosrulers.monitoring.rhobs
       version: v1
-  description: Observability operator to manage Monitoring, Logging, Tracing for easier
-    observability.
+  description: |2+
+
+    Observability Operator is an operator created in Go to setup and manage highly available Monitoring Stack using Prometheus and Thanos Querier.
```

Comment:
```suggestion
    Observability Operator is an operator created in Go to setup and manage highly available Prometheus monitoring stack.
```

Reply:
I felt its better to have description similar on line 44 and 97 since we didn't mention about Thanos querier earlier. I felt its bit confusing seeing two different descriptions. Leaving up to you which one to choose :)

Reply:
I will change the description (short) above to 
```
Observability Operator is an operator created in Go to setup and manage highly available Monitoring Stack using Prometheus and Thanos Querier
```
      A Go based operator to setup and manage highly available Monitoring Stack
      using Prometheus and Thanos Querier.
```
Both descriptions are different since they appear in different places on UI. The first one has to be concise.

Reply:
Use: https://operatorhub.io/preview to preview how OLM renders this.